### ISSUE TITLE
[BUG] stopAfterLastMessage not working

### ISSUE THREAD

**Issue by daavelar** [mateusjunges/laravel-kafka#120]
I'm calling the method stopAfterLastMessage() on the consumer building but the consumption don't stop.

I'm on ubuntu 22.04, running php 8.1 with rdkafka 6.0.3, laravel-kafka:1.8.1.

I check the code and in line 349 we have this:
`        if ($this->config->shouldStopAfterLastMessage() && RD_KAFKA_RESP_ERR__PARTITION_EOF === $message->err) {
`
RD_KAFKA_RESP_ERR__PARTITION_EOF equals -191 but $message->err is returning -185, i try to send a pull request but i get access denied, so i am sending the code that would be like this:

```
    private const CONSUMER_STOP_ERRORS = [
        RD_KAFKA_RESP_ERR__PARTITION_EOF,
        RD_KAFKA_RESP_ERR__TIMED_OUT
    ];
    
   ....
   
    if ($this->config->shouldStopAfterLastMessage() && in_array($message->err, self::CONSUMER_STOP_ERRORS)) {
        $this->stopConsume();
    }
```

**Comment by mateusjunges** [comment]
Hi @daavelar I'm returning from my vacation trip today, I'll check this issue tomorrow. Thanks for reporting.

**Comment by mateusjunges** [comment]
Hey @daavelar can you provide your consumer code so I can reproduce this issue?

**Comment by daavelar** [comment]
The documentation example it's good enough:

```
$consumer = \Junges\Kafka\Facades\Kafka::createConsumer(['topic'])
    ->withConsumerGroupId('group')
    ->stopAfterLastMessage()
    ->withHandler(new Handler)
    ->build();
```

**Comment by mateusjunges** [comment]
@daavelar can you try the #121 and let me know if it solves your issue please?

**Comment by daavelar** [comment]
Not working, still not entering on the if below, because RD_KAFKA_RESP_ERR__PARTITION_EOF = -191 and $message->err = -185

```
        if ($this->config->shouldStopAfterLastMessage() && RD_KAFKA_RESP_ERR__PARTITION_EOF === $message->err) {
            $this->stopConsume();
        }
```

**Comment by mateusjunges** [comment]
Thanks, I'll work on it today after office hour.

**Comment by mateusjunges** [comment]
Released on v1.8.3. Please let me know if it solves your problem.

**Comment by daavelar** [comment]
It Worked here! Tks!
